src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -7967,4 +7967,29 @@ public static boolean isSorted(boolean[] array) {
 
         return removeAll(array, Arrays.copyOf(indices, count));
     }
+
+    /**
+     * Returns an array containing the string representation of each entry in the argument array.
+     *
+     * <p>
+     * A {@code null} input Object[] returns {@code null}.
```

Comment:
One more thing came to my mind - we can document `null` elements handling. I don't have exact words for that, but something like `A {@code null} input Object[] returns {@code null}. {@code null} elements are preserved.`.